assets/src/js/admin/admin-settings.js

```diff
@@ -150,4 +150,127 @@ jQuery(document).ready(function ($) {
 			}
 		})
 	});
+
+	/**
+	 * Ajax call to get donation fields.
+	 */
+	$( '.give-export_donations #give-export_donations-form #give_form_for_csv_export' ).chosen().change( function () {
```

Comment:
@raftaar1191 please remove this from create a new JS file called `admin-exports.js`

Reply:
Okay @DevinWalker doing that